src/context/AppNav.tsx

```diff
@@ -26,7 +26,7 @@ export const AppNav = () => {
           component={ListTimeline}
           initialParams={{ displayName: "", handle: "" }}
         />
-        <Stack.Screen name={"compose"} component={Compose} />
+        <Stack.Screen name={"Compose"} component={Compose} />
```

Comment:
The `name` should be marked for internationalisation.

Reply:
done